README.md

```diff
@@ -1,87 +1,106 @@
-# Buildkite Test Splitter
+# Buildkite Test Engine Client
 
-Buildkite Test Splitter is an open source tool to orchestrate your test suites. It uses your Buildkite Test Analytics suite data to intelligently partition and parallelise your tests. 
-Test Splitter currently only supports RSpec but support for other frameworks is coming soon.
+Buildkite Test Engine Client (bktec) is an open source tool to orchestrate your test suites. It uses your Buildkite Test Engine suite data to intelligently partition and parallelise your tests.
+
+bktec supports RSpec and Jest.
 
 ## Migrating to 0.9.0
-Version 0.9.0 introduces a new feature that requires Test Splitter to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_SPLITTER_RESULT_PATH` environment variable. This variable specifies the location of where the runner should store test results.
+Version 0.9.0 introduces a new feature that requires bktec to read test results from the runner for retries and verification purposes. To enable this feature, it is necessary to configure the `BUILDKITE_SPLITTER_RESULT_PATH` environment variable. This variable specifies the location of where the runner should store test results.
```

Comment:
I think we can merge this to migrating to `1.0.0` section as the env var name is now also changed.